Dockerfile

```diff
@@ -172,6 +172,12 @@ EOF
 
 FROM dev AS development
 
+# TODO (SKH): This doesnt seem safe!
+RUN curl https://sh.rustup.rs -sSf | sh -s -- -y && \
+    source $HOME/.cargo/env && \
+    rustup default nightly
```

Comment:
this would be a great place for a multi-stage build. Can we have a separate rust base image that builds the libraries and then we just copy them over into the release container